scout/server/blueprints/variants/static/form_scripts.js

```diff
@@ -66,6 +78,27 @@ function validateForm(){
             return false;
         }
     }
+    // Validate Chromosome position form
+    //Expected format: <chr number>:<start>-<end>[+-]?<padding>
+    var chrom_pos = document.forms["filters_form"].elements["chrom_pos"].value.replaceAll(',', '')
+    const chromPosPattern = new RegExp("^([0-9]{1,2}|[X|T|MT]{1,2})(:([0-9]+)-([0-9]+)([+-]{1}[0-9]+)?)?$");
```

Comment:
As I mentioned before, the pattern could be improved by checking that chromosome range is [1-22] and X,Y,MT.
Also the suggestion from @dnil (allowing "chr" as prefix) could be implemented